packages/grafana-ui/src/components/Table/Table.story.tsx

```diff
@@ -105,8 +105,8 @@ export const BarGaugeCell = () => {
     {
       matcher: { id: FieldMatcherID.byName, options: 'Progress' },
       properties: [
-        { id: 'width', value: '200', isCustom: true },
-        { id: 'displayMode', value: 'gradient-gauge', isCustom: true },
+        { id: 'width', value: '200' },
+        { id: 'displayMode', value: 'gradient-gauge' },
```

Comment:
should the id's here not be "custom.width" if we remove isCustom?

Reply:
good catch